### PR TITLE
WRO-13689: Panel - fix console warnings

### PR DESCRIPTION
**Pull request by andreitirlalge**
### Checklist

* [x] I have read and understand the [contribution guide](http://enactjs.com/docs/developer-guide/contributing/)
* [ ] A [CHANGELOG entry](http://enactjs.com/docs/developer-guide/contributing/changelogs/) is included
* [ ] At least one test case is included for this feature or bug fix
* [x] Documentation was added or is not needed
* [x] Documentation was verified or is not changed
* [x] UI test was passed or is not needed
* [x] Screenshot test was verified or is not needed
* [ ] This is an API breaking change

### Issue Resolved / Feature Added
[//]: # (Describe the issue resolved or feature added by this pull request)
Fixed console warnings for Agate Panel by modifying storybook's ThemeEnvironment and Panel.

### Resolution
[//]: # (Does the code work as intended?)
[//]: # (What is the impact of this change and *why* was it made?)
The spotlightId and hideChildren warnings are fixed.

### Additional Considerations
[//]: # (How should the change be tested?)
[//]: # (Are there any outstanding questions?)
[//]: # (Were any side-effects caused by the change?)
Further investigation on how the changes made in storybook's ThemeEnvironment and Panel may affect the components.

### Links
[//]: # (Related issues, references)
WRO-13689

### Comments
Enact-DCO-1.0-Signed-off-by: Andrei Tirla andrei.tirla@lgepartner.com

### CODE REVIEW

File: samples/sampler/src/ThemeEnvironment/ThemeEnvironment.js [hunk 1]
```diff
@@ -182,6 +184,23 @@ const StorybookDecorator = (story, config) => {
...
+					<Cell><div style={{minHeight: ri.scaleToRem(399)}}>{sample}</div></Cell>
+				</SkinFrame>))}
+			</Scroller>
+		</Panel>) :
+		(<Scroller>{Object.keys(skins).map((skin) => (
+			<SkinFrame skin={skins[skin]} key={skin}>
+				<Cell size="20%" component={Heading}>{skin}</Cell>
+				<Cell>{sample}</Cell>
+			</SkinFrame>))}
+		</Scroller>);
```
Comment (stanca-pop-lgp) [inline samples/sampler/src/ThemeEnvironment/ThemeEnvironment.js:202]:
```suggestion
	const panelSample = (sample.type.displayName === 'Panel');

	const allSkinsSample =
		<Scroller>
			{Object.keys(skins).map((skin) => (
				<SkinFrame skin={skins[skin]} key={skin}>
					<Cell size="20%" component={Heading}>{skin}</Cell>
					<Cell style={{minHeight: panelSample ? ri.scaleToRem(300) : ''}}>{sample}</Cell>
				</SkinFrame>
			))}
		</Scroller>;

	const showAllSkinsSample = panelSample ? <Panel>{allSkinsSample}</Panel> : allSkinsSample;
```

File: samples/sampler/src/ThemeEnvironment/ThemeEnvironment.js [hunk 2]
```diff
@@ -193,15 +212,7 @@ const StorybookDecorator = (story, config) => {
...
-			{showAllSkins ?
-				<Scroller>
-					{Object.keys(skins).map((skin) => (
-						<SkinFrame skin={skins[skin]} key={skin}>
-							<Cell size="20%" component={Heading}>{skin}</Cell>
-							<Cell>{sample}</Cell>
-						</SkinFrame>
-					))}
-				</Scroller> : sample}
+			{showAllSkins ? allSkinsSample : sample}
```
Comment (stanca-pop-lgp) [inline samples/sampler/src/ThemeEnvironment/ThemeEnvironment.js:215]:
```suggestion
			{showAllSkins ? showAllSkinsSample : sample}
```

File: samples/sampler/src/ThemeEnvironment/ThemeEnvironment.js [hunk 3]
```diff
@@ -193,15 +212,7 @@ const StorybookDecorator = (story, config) => {
...
 		</Theme>
 	);
 };
```
Comment (stanca-pop-lgp) [inline samples/sampler/src/ThemeEnvironment/ThemeEnvironment.js:218]: Please remove empty line below.

File: samples/sampler/src/ThemeEnvironment/ThemeEnvironment.js [hunk 4]
```diff
@@ -3,7 +3,9 @@
 import kind from '@enact/core/kind';
 import PropTypes from 'prop-types';
 import {Row, Column, Cell} from '@enact/ui/Layout';
```
Comment (stanca-pop-lgp) [inline samples/sampler/src/ThemeEnvironment/ThemeEnvironment.js:5]:
```suggestion
import {Cell, Column, Row} from '@enact/ui/Layout';
```

File: samples/sampler/src/ThemeEnvironment/ThemeEnvironment.js [hunk 5]
```diff
@@ -3,7 +3,9 @@
...
+import ri from '@enact/ui/resolution';
 
+import Header from '@enact/agate/Header';
 import Heading from '@enact/agate/Heading';
 import {Panels, Panel} from '@enact/agate/Panels';
```
Comment (stanca-pop-lgp) [inline samples/sampler/src/ThemeEnvironment/ThemeEnvironment.js:10]:
```suggestion
import {Panel, Panels} from '@enact/agate/Panels';
```

File: samples/sampler/src/ThemeEnvironment/ThemeEnvironment.js [hunk 6]
```diff
@@ -3,7 +3,9 @@
 import kind from '@enact/core/kind';
 import PropTypes from 'prop-types';
 import {Row, Column, Cell} from '@enact/ui/Layout';
+import ri from '@enact/ui/resolution';
 
+import Header from '@enact/agate/Header';
```
Comment (stanca-pop-lgp) [inline samples/sampler/src/ThemeEnvironment/ThemeEnvironment.js:8]: Please remove this import if you apply my suggestions.

**Review by stanca-pop-lgp — CHANGES_REQUESTED** [review]

**Review by stanca-pop-lgp — APPROVED** [review]
LGTM

**Comment by jeonghee27** [comment]
When I test on my local env, the console error was deleted just by modifying the Panel.js file.
Why do we need to fix ThemeEnviroment.js?

**Comment by jeonghee27** [comment]
Please check Travis result

**Review by jeonghee27 — APPROVED** [review]
LGTM

**Pull request merged**